include/deal.II/base/aligned_vector.h

```diff
@@ -31,6 +31,8 @@
 #include <emmintrin.h>
```

Comment:
Oh yes, and you can remove <emmintrin.h> from here as well (mm_malloc.h was included via this file).